### DISCUSSION TITLE
Breadcrumbs

### DISCUSSION THREAD

**Discussion by neilg-dev** [oqtane/oqtane.framework#1792] [Ideas]
I use the breadcrumbs control in a custom theme, I've made a small adjustment so that it doesn't have an active link when IsClickable is disabled.  No doubt the code can be improved but I thought I'd share it in case it might be helpful as an idea?

@if (BreadCrumbPages.Any())
{
	<div class="b-header-divider">
        <nav class="nav" style="--bs-breadcrumb-divider: '>';" aria-label="breadcrumb">
        <ol class="breadcrumb">
            @foreach (var p in BreadCrumbPages)
            {
                if(p.IsClickable==false)
                    {
                    if (p.PageId == PageState.Page.PageId)
                    {
                        <li class="breadcrumb-item active">
                            @p.Name
                        </li>
                    }
                    else
                    {
                        <li class="breadcrumb-item">
                            @p.Name
                        </li>
                     }
                }
                else
                    {
                    if (p.PageId == PageState.Page.PageId)
                    {
                        <li class="breadcrumb-item active">
                            <a href="@NavigateUrl(p.Path)">@p.Name</a>
                        </li>
                    }
                    else
                    {
                        <li class="breadcrumb-item">
                            <a href="@NavigateUrl(p.Path)">@p.Name</a>
                        </li>
                    }
                }
            }
        </ol>
    </nav>
  </div>

}

Sample image for the theme, the navigation page "Samples" is not clickable

![image](https://user-images.githubusercontent.com/92930631/141353819-352dbb95-004f-4b4e-8a01-f65df86fed68.png)

**Comment by neilg-dev** [comment]
![image](https://user-images.githubusercontent.com/92930631/141354481-028ce195-97fb-42dd-8efa-153d90e55763.png)

**Reply by neilg-dev** [reply]
Should have previewed my initial submission as the code is mangled, have attached an image.

**Comment by leigh-pointer** [comment]
@NeilGallacherSQL I have created a Issue for this and also a fix on PR #1798
